lib/pkp

```diff
@@ -1 +1 @@
-Subproject commit d0e0deb561b459051083b4f0eac1d2562398d79d
+Subproject commit d071c7929e4dd54146b7953bae7666c93281cb04
```

Comment:
it is just to update the reference -- you already merged the PR -- the display of xml validation messages (https://github.com/pkp/pkp-lib/pull/1569)